lib/model/compose.dart

```diff
@@ -5,6 +5,28 @@ import 'internal_link.dart';
 import 'narrow.dart';
 import 'store.dart';
 
+/// The available user wildcard mention options,
+/// known to the server as [canonicalString].
+///
+/// See API docs:
+///   https://zulip.com/api/message-formatting#mentions-and-silent-mentions
+enum WildcardMentionOption {
+  all(canonicalString: 'all'),
+  everyone(canonicalString: 'everyone'),
+  channel(canonicalString: 'channel'),
+  // Deprecated in FL 247. Empirically, current servers (FL 330)
```

Comment:
needs a TODO-server comment (similar to https://github.com/zulip/zulip-flutter/pull/889#discussion_r1931389011 )